src/ucp/api/ucp.h

```diff
@@ -3311,13 +3318,18 @@ ucp_atomic_fetch_nb(ucp_ep_h ep, ucp_atomic_fetch_op_t opcode,
  * memory address @a remote_addr and the @ref ucp_rkey_h "remote memory handle"
  * @a rkey.
  * The routine is non-blocking and therefore returns immediately. However the
- * actual atomic operation may be delayed. The atomic operation is not considered complete
- * until the values in remote and local memory are completed.
- *
- * @note The user should not modify any part of the @a buffer or @a result after
- *       this operation is called, until the operation completes.
+ * actual atomic operation may be delayed. The atomic operation is not considered
+ * complete until the values in remote and local memory are completed. The
+ * result of the operation will be stored in the @a param->reply_buffer memory
+ * buffer. In the case of CSWAP the value in @a param->reply_buffer will be
+ * swapped into the @a remote_addr if the condition is true.
+ *
+ * @note The user should not modify any part of the @a buffer or
+ *       @a param->reply_buffer after this operation is called, until the
+ *       operation completes.
  * @note Only ucp_dt_make_config(4) and ucp_dt_make_contig(8) are supported in
  *       @a param->datatype, see @ref ucp_dt_make_contig
+ * @note This operation requires @a param->reply_buffer to be defined in @a param.
```

Comment:
`This operation requires @a param->reply_buffer and its corresponding flag @ref UCP_OP_ATTR_FIELD_REPLY_BUFFER   to be defined in @a param.`